### ISSUE TITLE
NPM3 Issue

### ISSUE THREAD

**Issue by xjamundx** [mzabriskie/rackt-cli#9]
```
~/dev/react-autocomplete (master) $ npm start

> react-autocomplete@0.1.4 start /Users/jamuferguson/dev/react-autocomplete
> rackt server

/Users/jamuferguson/dev/react-autocomplete/node_modules/rackt-cli/bin/../tasks/server: line 5: /Users/jamuferguson/dev/react-autocomplete/node_modules/rackt-cli/bin/..//node_modules/.bin/webpack-dev-server: No such file or directory

npm ERR! Darwin 14.5.0
npm ERR! argv "node" "/usr/local/bin/npm" "start"
npm ERR! node v0.12.6
npm ERR! npm  v3.1.0
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! react-autocomplete@0.1.4 start: `rackt server`
npm ERR! spawn ENOENT
npm ERR! 
npm ERR! Failed at the react-autocomplete@0.1.4 start script 'rackt server'.
npm ERR! This is most likely a problem with the react-autocomplete package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     rackt server
npm ERR! You can get their info via:
npm ERR!     npm owner ls react-autocomplete
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/jamuferguson/dev/react-autocomplete/npm-debug.log
```

I think this line breaks with NP3, because the node_modules are no longer nested
https://github.com/mzabriskie/rackt-cli/blob/master/tasks/server#L3

And even after fixing that one a few other places break as well. For now I can re-install with npm2 and it's all good....

**Comment by mzabriskie** [comment]
I haven't installed npm3 yet. Let me give it a try, and see how to go about supporting both.

**Comment by ikhthiandor** [comment]
I am also facing the same issue. Any suggested fix?

**Comment by mzabriskie** [comment]
Should be fixed by #25
